doc/source/upgrading_guide.rst

```diff
@@ -34,6 +34,21 @@ of incompatible changes that could cause automatic upgrade to fail:
    With partitioned tables that use the TokuDB or MyRocks storage
    engine, the upgrade only works with native partitioning.
 
+.. note::
+
+    :ref:`encrypting-threads` is an **EXPERIMENTAL** feature.
+    
+    Before encryption is set to "ON", the user should consult the
+    INFORMATION_SCHEMA_ENCRYPTION table to verify which tables are excluded from
```

Comment:
s/INFORMATION_SCHEMA_ENCRYPTION/INFORMATION_SCHEMA.INNODB_TABLESPACES_ENCRYPTION/

Reply:
The tablespaces that are excluded from encryption threads have 'Y' in EXCLUDED column.

Reply:
Rewritten.